nginx.conf

```diff
@@ -98,7 +98,7 @@ http {
       
       # Match paths with backslashes at site root or after
       location ~ ^/[^/]+/[^/]+/[^/]+/.*[\\\]+ {
-        return 404 "Not Found";
+        rewrite ^(.*)$ @notfound;
```

Comment:
I would like to understand why this is better, but I have no objection to it.

Reply:
I could have either added the content-type header or this. My hope was that this would result in returning the site's actual 404, but it still seems to be returning the openresty 404